tests/switchScrollingEffect.test.ts

```diff
@@ -36,7 +36,7 @@ describe('switchScrollingEffect', () => {
 
     switchScrollingEffect();
 
-    expect(document.body.style.cssText).toBe('position: relative;');
+    expect(document.body.style.cssText).toBe('position: relative; width: calc(100% - 20px);');
```

Comment:
Why add this?

Reply:
seems that I have a different env from CI. reverted.

Reply:
not sure whether comes from here: https://github.com/react-component/util/blob/43cf427a3aeb15c70c2588e1defec8d99d281cfd/src/switchScrollingEffect.js#L39-L42